src/core/src/util.cpp

```diff
@@ -82,9 +82,9 @@ size_t ngraph::hash_combine(const std::vector<size_t>& list) {
 }
 
 void* ngraph::ngraph_malloc(size_t size) {
```

Comment:
```suggestion
void* ngraph::ngraph_calloc(size_t size) {
```

Reply:
It is not a backward compatible change. Moreover this API is deprecated. Let's keep it as is.

Reply:
this is `util` routine. Does it really affect backward-compatibility?

Reply:
Yes it is our public API

Reply:
I see. May be it makes sense to create new routine then and to leave this one as is?

Reply:
Let's just remove this method in 2024.0.
We don't need to introduce the new API which will be removed soon, we don't use this method. So let's avoid additional steps and changes.